packages/wrangler/templates/first-party-worker-module-facade.ts

```diff
@@ -11,8 +9,11 @@ type Env = {
  * Setup globals/vars as required
  */
 
-export default {
+export default <ExportedHandler>{
 	async fetch(request: Request, env: Env, ctx: ExecutionContext) {
+		if (worker.fetch === undefined) {
+			throw new TypeError("Entry point missing `fetch` handler");
+		}
```

Comment:
What about FPWs that only export scheduled handlers?

Reply:
Yes, that's true. That's made me wonder though—should this file not also export `scheduled`, `queue`, etc... events for FPWs?